easybuild/easyconfigs/c/CGAL/CGAL-4.6-intel-2015a-GLib-2.44.1-Python-2.7.10.eb

```diff
@@ -0,0 +1,31 @@
+name = 'CGAL'
+version = '4.6'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'strict': True}
+
+# note: source URL needs to be updated for a new version (checksums too)!
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/34703/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['fd85f93b2d666970bccda2c55d20a231']
+
+pysuff = '-Python-2.7.10'
+versionsuffix = '%s%s' % ('-GLib-2.44.1', pysuff)
+
+dependencies = [
+#GMP does not compile correctly with ICC on Haswell architecture
+    ('GMP', '6.0.0a', '', ('GCC', '4.9.2')),
```

Comment:
maybe only for `intel/2015b`, cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/g/GMP/GMP-6.0.0a-intel-2015b.eb

@hajgato: good to leave as is, just the indent for the comment, and mention that it doesn't work for the icc in intel/2015a specifically